src/components/task-plan/homework.cjsx

```diff
@@ -184,6 +198,26 @@ AddExercises = React.createClass
 
 
 ExerciseSummary = React.createClass
+  componentDidMount: () ->
+    el = @getDOMNode()
+    @staticPosition = el.getBoundingClientRect().top - document.body.getBoundingClientRect().top
+    @handleScroll() # Update scroll position immediately on mount
+    @optimizedScrollFunc = throttle('scroll', 'optimizedScroll')
```

Comment:
This is attaching an event listener to the DOM element, which must exist (been mounted) for it to attach correctly.  Defining it on the component directly does not guarantee the element exists in the DOM (it likely does not).